packages/frontend/src/lib/sanity.ts

```diff
@@ -114,7 +114,8 @@ export async function getArticlesFromTag(tagSlugName: string): Promise<Article[]
 			date,
 			authors[]->{name},
 			slug,
-			category->
+			category->,
+			media
 
```

Comment:
@monkeyjonjon make sure to just format these `groq` queries and remove spaces, to keep the code clean and tidy. Just something small, but makes a difference in [git diffs](https://www.git-tower.com/learn/git/faq/git-diff), so no whitespaces are there in code comparisons (this is why we use prettier to format our code with husky precommit). Don't worry, as I make this mistake too.